main.go

```diff
@@ -16,84 +19,64 @@ import (
 )
 
 func main() {
-	log.SetFlags(log.LstdFlags | log.Lshortfile)
+	go func() {
```

Comment:
Perhaps we can add a new flag, like `-pprof` for running this

Reply:
Hehe... I forgot to clean up. Let me fix that.

Reply:
We can have a different main target for it, separate files and all.